webview-ui/src/components/asset/AssetGeneral.vue

```diff
@@ -512,6 +520,19 @@ function runAssetWithDownstream() {
   });
 }
 
+
+
+function runPipelineWithContinue() {
+  let payload = getCheckboxChangePayload();
+  payload = payload + " --downstream" + " --environment " + selectedEnv.value + " --continue";
```

Comment:
why do we need these values for `--continue`, shouldn't it be getting it from the previous state?